assets/sass/kerrokantasi/_commentlist.scss

```diff
@@ -1,3 +1,12 @@
+@keyframes highlightComment {
+    from {
+        background-color: $hel-engel;
```

Comment:
There should be no Helsinki specific variables used in the `kerrokantasi/` styles. Helsinki specific styling should be added under the `hel/` folder. This variable can be changed into `$theme-primary-light` here and these styles can be overridden inside `hel/` styles folder if needed.

Reply:
replaced by theme-primary-light